docs/player-support/google-cast.md

```diff
@@ -5,13 +5,14 @@ Music Assistant has full support for Google Cast based devices. This includes Go
 ## Features
 
 - Cast speakers are auto detected by Music Assistant
-- Music Assistant supports playing to cast groups which are created in the Google Home app.
-- When using Google cast groups then perfect sync across players in that group is possible.
-- Any physical control buttons on the device should be supported as well as voice control.
+- Music Assistant supports playing to cast groups which are created in the Google Home app
+- When using Google cast groups then perfect sync across players in that group is possible
+- Any physical control buttons on the device should be supported as well as voice control
 
 ## Known Issues / Notes
 
 - Cast speakers do not support crossfading of audio. If you want crossfade and/or full gapless support, enable the "[flow mode](../faq/normalization.md/#track-queueing)" in the player's advanced settings. Enabling flow mode may solve playback issues however it might come with the side effect of disabling actual physical buttons and/or display of metadata on the device itself
-- If your Chromecast speakers are not auto detected or randomly unavailable then make sure that your Cast enabled speakers are on the same network/subnet as your Music Assistant server. Additionally, ensure that multicast traffic (more specifically mDNS) can travel freely as that is used for the discovery of players.
-- After re-enabling a disabled speaker, it can take a while before the speaker is rediscovered, the process can be sped up by restarting Music Assistant.
-- It is not possible to group/sync cast players from within the MA interface (simply because an api does not exist for that).
+- If your Chromecast speakers are not auto detected or randomly unavailable then make sure that your Cast enabled speakers are on the same network/subnet as your Music Assistant server. Additionally, ensure that multicast traffic (more specifically mDNS) can travel freely as that is used for the discovery of players
+- After re-enabling a disabled speaker, it can take a while before the speaker is rediscovered, the process can be sped up by restarting Music Assistant
+- It is not possible to group/sync cast players from within the MA interface (simply because an api does not exist for that)
+- Google TV devices (not the AV dongles) are disabled by default
```

Comment:
as well as actual TV's - just anything that reports its model as a tv device

Reply:
I thought that and thus "Google TV devices" was my attempt to capture that idea. Thoughts on better words?

Reply:
Well the name of their latest stick is literally Google TV so maybe just refer to TV/Video devices here ?

Reply:
Ok done